docker/Dockerfile

```diff
@@ -249,7 +249,7 @@ RUN install-cmake-from-source https://github.com/libuv/libuv/archive/v1.42.0.tar
     -DBUILD_TESTING=OFF
 
 # NUClear!
-RUN install-from-source https://github.com/Fastcode/NUClear/archive/f4a60c7269e277a504635a603c2890fb1a70289d.tar.gz \
+RUN install-from-source https://github.com/Fastcode/NUClear/archive/304ff3b5e21b5bf7ea222f5cf1adca7622036f2a.tar.gz \
```

Comment:
I can't find this commit on the repo (although it exists), what is the purpos (and also should we be trying to bring our NUClear version up to date?)

Reply:
github says it's not on a branch 😳

Reply:
It's main, just one commit ahead on main from what you have. The bug I fixed doesn't influence anything else so updating isn't a matter of urgency (except for this)

Reply:
https://github.com/Fastcode/NUClear/commit/b36b8731af9bde5742ece832d0a3c0f317ac6e6e
this is the next commit after on main?
If you got to it https://github.com/Fastcode/NUClear/commit/304ff3b5e21b5bf7ea222f5cf1adca7622036f2a
"This commit does not belong to any branch on this repository, and may belong to a fork outside of the repository."

Reply:
Hmm, was supposed to include this PR, maybe I got the hash wrong https://github.com/Fastcode/NUClear/pull/57

Reply:
Seems I picked a commit on that PR rather than the one on main after it merged